### ISSUE TITLE
Broken Icons in toolbar - macOS/Chrome

### ISSUE THREAD

**Issue by Twista** [Leaflet/Leaflet.draw#629]
Hello, first - thanks for awesome work! 

There is little bug i found.

## How to reproduce

- Leaflet version I'm using: 1.0.1
- Leaflet Draw version I'm using:  ^0.4.0 or master (`be0ca82`)
- Browser (with version) I'm using: Chrome
- OS/Platform (with version) I'm using: macOS El Capitan

The easiest way how to reproduce is clone current master and open example from docs

```
$ git clone https://github.com/Leaflet/Leaflet.draw.git
$ open Leaflet.draw/docs/examples/full.html
```

Then use zoom tool in browser - first icon in every bar has broken icons

## What behaviour I'm expecting and which behaviour I'm seeing

![leaflet_draw_vector_editing_handlers](https://cloud.githubusercontent.com/assets/1297511/20136093/60185e90-a673-11e6-8d0d-f1ffbe63d9d8.png)

Expecting behaviour is that icons are not broken.

## Notes

Tried on multiple computers with macOS - The problem seems to be only in Chrome (FF and Safari are ok).

Find out the problem is in CSS (`leaflet.draw.css:25`) and only if browser uses `images/spritesheet.svg `. When i changed styles to use `images/spritesheet.png` - problem disappeared

I would love to send pull-request, but i'm missing experience working with SVG

Thanks!

edit: formating

**Comment by ddproxy** [comment]
Thanks! I'll include this in the patch later tonight.

**Comment by Twista** [comment]
Awesome, thank you!

**Comment by ddproxy** [comment]
@Twista - identified the issue as a chrome bug.

Patch goes out tonight to fix this for Leaflet.Draw, PR Leaflet/Leaflet#5085 on Leaflet for fixing this within Leaflet.

An issue has been reported to Chrome linking to this ticket as well as how to reproduce.

**Comment by ddproxy** [comment]
@Twista Please close once you have confirmed the issue is resolved, 0.4.3 has been released.

**Comment by Twista** [comment]
Just tried and works perfectly. Awesome - Thank you for very quick fix!
